cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,74 @@
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
```

Comment:
This cmake_uninstall.cmake.in file should be moved to the cmake folder and renamed to Uninstall.cmake.in to conform with their naming style.